.github/workflows/lint-commits.yml

```
@@ -0,0 +1,22 @@
+name: Lint Commit Messages
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)